### DISCUSSION TITLE
Connection to Wallet Loading Forever

### DISCUSSION THREAD

**Discussion by ceoforex1** [Chia-Network/chia-blockchain#8707] [Support]
Connection to wallet loading screen taking forever, i uninstalled and reinstalled, same problem, what should i do?

**Comment by loppefaaret** [comment]
It can take some minutes to load up the blockchain database, it is really depending on what hardware you got Chia installed on ? both the CPU, the amount of memory, and the drive it is stored on can all have an effect on it

**Comment by BrandtH22** [comment]
@ceoforex1 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
